kong/db/dao/targets.lua

```diff
@@ -111,7 +115,7 @@ function _TARGETS:delete(pk)
     return nil, err, err_t
   end
 
-  return self:insert({
+  self:insert({
```

Comment:
Should we handle possible errors in this case?

Reply:
fixed!